.github/automation/performance/run_benchdnn_compare.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+
+# *******************************************************************************
+# Copyright 2025 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+#
+# Runs a user-supplied benchdnn command on two oneDNN binaries.
+# Usage
+#   .github/automation/performance/run_benchdnn_compare.sh \
+#       ${BASE_BIN} ${NEW_BIN} ${BASE_OUT} ${NEW_OUT} ${CMD}
+
+set -euo pipefail
+
+BASE_BINARY=$1
+NEW_BINARY=$2
+BASE_OUT=$3
+NEW_OUT=$4
+CMD_STR=("${@:5}")
```

Comment:
Another possible cause of your parsing issues is that you are creating an array here since you have surrounded `${@:5}` with round brackets. Again, it is a little difficult to figure out what your original intent was. Based on the name of the variable I assume you just want a plain string; in which case please remove the parenthesis.